cmd/spire-server/cli/entry/delete.go

```diff
@@ -26,9 +28,10 @@ func newDeleteCommand(env *commoncli.Env) cli.Command {
 
 type deleteCommand struct {
 	// ID of the record to delete
-	entryID string
-	env     *commoncli.Env
-	printer cliprinter.Printer
+	entriesIDs StringsFlag
```

Comment:
nit, this should probably be
```suggestion
	entryIDs StringsFlag
```